### PR TITLE
Adds a mmm snow endpoint to access historical and modeled snowfall equivalent.

### PR DESCRIPTION
**Pull request by charparr**
This PR creates a new endpoint under the top-level `mmm` route to access modeled snowfall equivalent data (SFE) for ultimate use in the Arctic-EDS. The routing follows the existing convention for summarizing over either `historical` or `projected` or both (`hp`) - or for accessing `all` data values. The `all` option is also available for CSV export. The native temporal resolution of this dataset is decadal, so custom time-slicing is *not* available in this endpoint because users could unwittingly summarize over very few values. For example, a 2020-2050 summary would only compute min-mean-max over 3 values - not 30.

To use this endpoint fire up the API and point it at Apollo
`export API_RAS_BASE_URL=http://apollo.snap.uaf.edu:8080/rasdaman/`

Some URLs to test:
http://localhost:5000/mmm/snow/snowfallequivalent/historical/61.5/-147
http://localhost:5000/mmm/snow/snowfallequivalent/projected/61.5/-147
http://localhost:5000/mmm/snow/snowfallequivalent/hp/61.5/-147
http://localhost:5000/mmm/snow/snowfallequivalent/all/61.5/-147

A CSV export to test:
http://localhost:5000/mmm/snow/snowfallequivalent/all/61.5/-147?format=csv

The above URLs should return integer values of SWE (units are inches of water) and the `hp` and `all` responses should reveal a decreasing trend in SFE which is expected as the length of snow season shortens in response to a warming climate.

A malformed summary option will 400:
http://localhost:5000/mmm/snow/snowfallequivalent/foo/61.5/-147

While a request outside the data boundary will 404:
http://localhost:5000/mmm/snow/snowfallequivalent/hp/51.5/-147

Notes
 - If no summary option is given `hp` is used as a default, e.g.,:
http://localhost:5000/mmm/snow/snowfallequivalent/61.5/-147
 - For low SFE (dry) areas, there is a signal-to-noise issue with this dataset, e.g.,:
 http://localhost:5000/mmm/snow/snowfallequivalent/65.5/-146

### CODE REVIEW

File: routes/snow.py
Comment (charparr) [reply routes/snow.py:101]: Having a digit in the tenths place for inches of water makes sense to me! So before we merge this should I go ahead and re-process and re-ingest the snowfall coverage as described (XREF https://github.com/ua-snap/rasdaman-ingest/issues/34) so we can get one decimal place over here in the API, or should I "fake it" for now for the min and max (e.g, just add `.0` on the end so that a min of `4` becomes `4.0`) and then compute the mean with one decimal?

Comment (kyleredilla) [reply routes/snow.py:101]: I think just changing lines 101 and 113 to use the `round` function instead of `int` would be sufficient. The min/max values should already be good to go for when the underlying values are updated I think!

**Review by kyleredilla — APPROVED** [review]
Look good!

**Pull request merged**
